packages/vaadin-grid/src/vaadin-grid-scroller.d.ts

```diff
@@ -20,7 +20,7 @@ declare class ScrollerElement extends PolymerIronList {
 
   _createScrollerRows(): void;
 
-  _canPopulate(): void;
+  _canPopulate(): boolean;
```

Comment:
Changed to align with another type. This was probably an error, the method returns a boolean.